akka-remote/src/test/scala/akka/remote/classic/ActorsLeakSpec.scala

```diff
@@ -209,9 +209,7 @@ class ActorsLeakSpec extends AkkaSpec(ActorsLeakSpec.config) with ImplicitSender
         remoteSystem.terminate()
       }
 
-      EventFilter.warning(pattern = "Association with remote system", occurrences = 1).intercept {
-        Await.ready(remoteSystem.whenTerminated, 10.seconds)
-      }
+      Await.ready(remoteSystem.whenTerminated, 10.seconds)
```

Comment:
are we sure we want to use `Await.ready` and not `Await.result` here? `ready` will not throw

there is another `ready` earlier in this test